defaults/main.yml

```diff
@@ -14,14 +14,27 @@ retry_max_attempts: 5
 zookeeper_notification_path: /notification/alarms
 zookeeper_notification_retry_path: /notification/retry
 monasca_conf_dir: /etc/monasca
-monasca_log_dir: /var/log/monasca
 monasca_log_level: WARN
+monasca_notification_name: monasca-notification
 pip_conf_dir: ~/.pip
-mysql_db: mon
+database_support: true
```

Comment:
Why have this variable? The monasca-notification won't run without a db so this will always be true.